cookbooks/swift/recipes/setup.rb

```diff
@@ -239,6 +239,30 @@
   creates "/etc/bash_completion.d/aws_bash_completer"
 end
 
+# rclone setup
+# ~/.config/rclone/rclone.conf
+directory "/home/#{node['username']}/.config" do
+  owner node['username']
+  group node['username']
+  mode 0700
+  action :create
+end
+directory "/home/#{node['username']}/.config/rclone" do
+  owner node['username']
+  group node['username']
+  mode 0700
+  action :create
```

Comment:
nit: Could add a `recursive true` here and skip the parent dir above.

Reply:
great idea! My chef-foo is a little rusty :)